src/components/HelloWorld.vue

```diff
@@ -1,17 +1,21 @@
 <script setup lang="ts">
-import { ref } from 'vue';
+import { ref, inject } from 'vue';
 import { Button as WikitButton } from '@wmde/wikit-vue-components';
+import Messages, { MessagesKey } from '@/plugins/MessagesPlugin/Messages';
 
 defineProps<{ msg: string }>();
 
 const count = ref( 0 );
+const $messages = inject( MessagesKey, new Messages() );
 </script>
 
 <template>
 	<h1>{{ msg }}</h1>
 
 	<p>See <code>README.md</code> for more information.</p>
 
+	<p>i18n example: {{ $messages.get( $messages.KEYS.LEMMA_INPUT_LABEL ) }}</p>
```

Comment:
This discussion doesn’t have to block merging, but I wonder what the benefit of the `KEYS` enum is here? I think it would be easier to find the message used here in `i18n/en.json`, or to find all the places in the code using a certain message, if we removed this level of indirection and just had:

```html
 	<p>i18n example: {{ $messages.get( 'wikibaselexeme-newlexeme-lemma' ) }}</p>
```

We could still have tsc check that only known message keys are used by declaring the message key as `keyof MessageKeys` or something like that.

Reply:
Yes, I like that more, and we even still have IDE autocompletion! 🙌